### PR TITLE
Updated installation instructions

### PR DESCRIPTION
**Pull request by fonfon**
I'm not too familiar with freedom-maker, so please correct my mistakes, I just found the documentation to be a bit out of date.
Is http://anonscm.debian.org/git/freedombox/freedom-maker.git still the preferred source for freedom-maker, or should we point users to github?

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -313,8 +308,8 @@ Required for DreamPlug:
 
 Now, fetch the git source of freedom-maker and build the image you want:
 
-    $ git clone http://anonscm.debian.org/git/freedombox/freedom-maker.git freedom-maker
-    $ make -C freedom-maker dreamplug-image raspberry-image virtualbox-image
+    $ git clone https://github.com/freedombox/freedom-maker.git freedom-maker
+    $ make -C freedom-maker dreamplug raspberry virtualbox
```
Comment (SunilMohanAdapa) [inline README]: I think Alioth repository is still the primary repository.  GitHub pull requests were being accepted, however.  Both the repositories were up-to-date last I checked.  @petterreinholdtsen would know.

According to targets inside the Makefile, this command is correct:
```sh
$ make -C freedom-maker dreamplug-image raspberry-image virtualbox-image
```

Comment (petterreinholdtsen) [inline README]: [Sunil Mohan Adapa]
> I think Alioth repository is still the primary repository.  GitHub
> pull requests were being accepted, however.  Both the repositories
> were up-to-date last I checked.  @petterreinholdtsen would know.

Before the freedombox project on github, I considered Alioth the
authorative source for freedom-maker, and my private copy a convenience
for people to send pull requests to.

I have not considered what we should consider the authorative source
now, but will keep using Alioth as the primary repo until someone
protests. :)

-- 
Happy hacking
Petter Reinholdtsen

**Comment by fonfon** [comment]
@SunilMohanAdapa Could you point out the lines that suggest that `virtualbox-image` is the correct target?
Trying it on the commandline, `virtualbox` works (well, it fails but due to a missing dependency to be precise) - but `virtualbox-image` doesn't work:

    flipper@flipper:~/code$ make -C freedom-maker virtualbox-image
    make: Entering directory '/home/flipper/code/freedom-maker'
    make: *** No rule to make target 'virtualbox-image'.  Stop.

**Comment by SunilMohanAdapa** [comment]
On Tuesday 13 January 2015 07:40 PM, fonfon wrote:
> @SunilMohanAdapa Could you point out the lines that suggest that `virtualbox-image` is the correct target?
> Trying it on the commandline, `virtualbox` works (well, it fails but due to a missing dependency to be precise) - but `virtualbox-image` doesn't work:
> 
>     flipper@flipper:~/code$ make -C freedom-maker virtualbox-image
>     make: Entering directory '/home/flipper/code/freedom-maker'
>     make: *** No rule to make target 'virtualbox-image'.  Stop.

Sorry, I was looking at an older tree.  Your command is correct.

-- 
Sunil

**Pull request merged**